css/default.css

```diff
@@ -92,3 +92,10 @@ td.print-overdue    { font-weight: bold; }
 .table-nonfluid {
    width: auto !important;
 }
+
+/* Markdown */
+.markdown-body blockquote {
```

Comment:
> There will be a lot of places where markdown will be rendered

Like how many places do you think? 10 would be sound reasonable.

That way would make changes in design easier without affecting if admins customize the CSS should they want later.